### PR TITLE
refactor(db, webserver): Refactor InvitationDao to use query_as!

### PR DESCRIPTION
**Pull request by boxbeam**

### CODE REVIEW

File: ee/tabby-webserver/src/service/auth.rs [hunk 1]
```diff
@@ -273,7 +273,7 @@ impl AuthenticationService for AuthenticationServiceImpl {
     }
 
     async fn delete_invitation(&self, id: &ID) -> Result<ID> {
-        Ok(self.db.delete_invitation(id.as_rowid()?).await?.as_id())
+        Ok((self.db.delete_invitation(id.as_rowid()? as i64).await? as i32).as_id())
```
[the remark this answers is not in the document]
Comment (boxbeam) [reply ee/tabby-webserver/src/service/auth.rs:276]: That PR isn't merged yet, so I'll have to wait on that

[the remark this answers is not in the document]
Comment (wsxiaoys) [reply ee/tabby-webserver/src/service/auth.rs:276]: kk, feel free to add trait in this PR - I can resolve conflict later.

**Review by wsxiaoys — COMMENTED** [review]
LGTM - will hold till 0.9rc.1

**Comment by wsxiaoys** [comment]
Please rebase and

**Comment by wsxiaoys** [comment]
and fix tests?

**Comment by wsxiaoys** [comment]
related TAB-521

**Pull request merged**